src/main/java/com/github/netelli/ProductsDAO.java

```diff
@@ -29,12 +32,14 @@ public void init() {
 
     public void deleteData() throws SQLException {
         try (Statement statement = connection.createStatement()) {
+            logger.info("Delete items from 'products' with brandId 2");
```

Comment:
1) All messages about start of some action should be placed at the very beginning of method, so if createStatement() will throw error you will have a hint in log about where something went wrong.

2) if you print some specific information about query (like "products' with brandId 2") it's better to assign value of brandId to variable and use it in multiple places, so won't have to look after all the code to implement changes (in case you will decide to change brandId from 2 to 5 for example);